### ISSUE TITLE
"R_FlatNumForName: not found" error (but this time on a real MEDPro)

### ISSUE THREAD

**Issue by rfmerrill** [viciious/d32xr#83]
Same behavior as #72  but I am using a genuine Mega Everdrive Pro.

This is on 3.1, both the prebuilt from cdromance and one I made using xdelta from romhacking.

I only own one 32X, which is a Japanese one.

Hardware combos I have tried:
- Japanese model 1 MD VA6 with or without a Japanese model 1 mega-CD (early revision). The VA6 has a region switch and I have tried both Japan and US region. The Mega-CD has Leo Olivera's patched version of the Japanese BIOS
- US model 2 Genesis with a US model 2 sega CD (Funai). Again with a patched BIOS.

I am using a single six-button controller

I get identical behavior no matter what combo I use--error in the title right after the credits screen. I have not tried my other MEDPro, or a stock CD bios (I'd have to pull a chip and swap to try that and I get the same error without a sega CD at all).

**Comment by rfmerrill** [comment]
2.2b works fine by the way, as do other 32x games.

**Comment by viciious** [comment]
All I know for sure is that the software works on legitimate MedPro's. It is not a software issue.

**Comment by rfmerrill** [comment]
I wonder what the issue could be then. Has anyone successfully tested on a Japanese 32x?

Is it possible my medpro firmware is out of date? Maybe something in my settings?

**Comment by viciious** [comment]
The Japanese 32X has got nothing to do with the problem, that's for sure.
You'd always test on the latest firmware of the flashcart though.

**Comment by rfmerrill** [comment]
Yep, my everdrive firmware was old. Sorry to waste your time! I guess I assumed someone must have run into this problem already if that were the case but my searches brought up nothing. At least this issue is here now if someone else runs into it!

By the way there was an additional symptom: The DOOM logo was missing from the title splash, it just had the flames.
